### ISSUE TITLE
sulu logo

### ISSUE THREAD

**Issue by Doum-Super** [sulu-io/sulu-standard#615]
Hello, how can i modify sulu logo in admin and browser link ?

**Comment by danrot** [comment]
There is a favicon placed in the `web` folder, I guess that's what you mean by browser link.

And I am afraid to tell you, that it is currently not possible to modifiy the sulu logo in the admin...

**Comment by stefann-sasori** [comment]
@Doum-Super
 Maybe this could help [http://blog.sulu.io/how-to-develop-a-bundle-in-the-sulu-admin-2](url) ;

As sulu logo is set in admin via css; you could follow the above link and add your own css to the admin (via a simple admin bundle) that would overwrite the logo image url.

Something like 
```javascript
// file: Resources/public/js/main.js
require.config({
    paths: {
//        examplenews: '../../examplenews/js',
        examplenewscss: '../../examplenews/css'
    }
});

```
And then in examplenews/css/style.css

```css
.logo{
    background-img: url('path/to/your/logo.png') !important;
}
```
However I don't know if this is ok with Sulu philosophy. It looks hacky. @danrot could give his view.

**Comment by danrot** [comment]
@stefann-sasori Yeah, that solution would probably work, but it is indeed quite hacky, and not what this mechanism is thought for... We would have to introduce a configuration to set the path to the logo.
